case-studies/wso2.md

```diff
@@ -26,7 +26,7 @@ Previously, WSO2 had many manual operations involved in its internal application
 
 WSO2 initiated a strategic plan to modernize its applications, migrate to cloud-based solutions, and introduce automation for repetitive tasks using Ballerina services and connectors as illustrated in the diagram below. The backend of the apps was written in Ballerina, the front end was developed using React, and they were deployed in [WSO2 Choreo](https://wso2.com/choreo/).
 
-![WSO2 vs. Ballerina](/images/case-studies/wso2-vs-ballerina.png)
+<img src="/images/case-studies/wso2-vs-ballerina.png" alt="WSO2 vs. Ballerina">
```

Comment:
We decided not to use titles like `WSO2 vs. Ballerina`, right? It should be just `WSO2`. `vs.` isn't ideal wording for this.

Reply:
Updated.